### PR TITLE
Change image tag to v2

### PR DESCRIPTION
**Pull request by sq2gxo**

### CODE REVIEW

File: build/docker-deploy.sh [hunk 1]
```diff
@@ -11,9 +11,9 @@ if [[ "${PUSH:-}" == "true" ]]; then
     base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
 
     ./build/version.sh
-    docker tag dynatrace/dynatrace-gcp-function:latest dynatrace/dynatrace-gcp-function:$TAG
+    docker tag dynatrace/dynatrace-gcp-function:v2-latest dynatrace/dynatrace-gcp-function:$TAG
```
Comment (karolkieglerski) [inline build/docker-deploy.sh:14]: consider to not change it to `v1-latest` because we are moving from `0.x` to `1.x`

Comment (dtwwolinski) [reply build/docker-deploy.sh]: Shouldn't it be more like ```dynatrace/dynatrace-gcp-function/v1:latest``` ?

Comment (sq2gxo) [reply build/docker-deploy.sh]: No, image name should not change. It's just the tag change from "latest" to v1-latest"

**Review by karolkieglerski — APPROVED** [review]

**Comment by sq2gxo** [comment]
Confirmed that both tags (v1-latest and tag) should work correctly by pushing image with two tags to gcr.io
image

File: build/docker-deploy.sh [hunk 2]
```diff
@@ -11,9 +11,9 @@ if [[ "${PUSH:-}" == "true" ]]; then
     base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
 
     ./build/version.sh
```
Comment (karolkieglerski) [inline build/docker-deploy.sh:13]: please move it the same as on master

**Pull request merged**